iphone/metadata/en-US/subtitle.txt

```diff
@@ -1 +1 @@
-Trails, Travel, GPS Navigation
+Navigate with Privacy
```

Comment:
@oleg-pah I have doubts that removal of important keywords here will help with ASO, but let's try once.